### DISCUSSION TITLE
[Feature] Send back pictures to the "Unclassified pool" rather than creating a new Person

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6007] [Feature Request]
### The feature

When reassigning a face, the only options are to chose an existing Person or to create a new one.
This is ok when the current cluster is only about the same person. But it is wrong when multiple persons appear to be wrongly clustered together.

Feature:
A third option: to send back the picture to the clustering algorithm. If a user has been pro-actively naming people and fixing misclassification, this would help find the right clusters and avoid the user to manually pick the correct person asset after asset.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by aisbergde** [comment]
this would be very helpful. And there could be one more option: remove from cluster and hide. This would require less clicks to just remove wrong assignments.

**Comment by bw1faeh0** [comment]
I need such a feature!
I have wrongly assigned faces to a person, which I don’t know, so I want to un-assign the face without creating new person (I don’t know…).

**Comment by bo0tzz** [comment]
#5276
